html/sales.html

```diff
@@ -8,13 +8,56 @@
     <body>
 
       <header>
-        <h1>COME GET YOUR SALMON COOKIES!!!</h1>
+        <h1>Awww Yissss...Motha...fu#^in'...Salmon Cookies</h1>
       </header>
 
       <figure>
         <img class="fishPicture" src="../img/salmon.png" alt="">
      </figure>
 
+     <section id="section1">
+
+     </section>
+
+     <section id="section2">
+
+       <form id="newStoreForm" action="sales.html" method="get">
+
+         <fieldset class="formField">
+
+         <ul>
```

Comment:
This is contained within the `fieldset` element and so should be indented one more level